include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -202,6 +202,8 @@ make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
             // defined later since we need openPMDWriter constructor
 
             plugins::multi::Option<std::string> notifyPeriod = {"period", "enable openPMD IO [for each n-th step]"};
+            plugins::multi::Option<std::string> range
+                = {"range", "define the output range in cells for each dimension e.g. 1:10,:,42:"};
```

Comment:
I think you forgot to add it to `TBG_macros.cfg` (which is imo quite a bad workflow that we have to add any option to multiple places, but at least that's the current workflow)